<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paraDialog</class>
 <widget class="QDialog" name="paraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_0">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>参考信息</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>单次数据量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>总数据量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>预估探测时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_totalsize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>111</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>探测方式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sglfilesize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>111</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_totalTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>71</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_detectDir">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>31</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>111</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> M</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>111</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> M</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>280</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>激光参数</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_laserPulseWidth">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>激光重频：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_laserRPF">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>激光波长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_laserWaveLength">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> nm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>脉冲宽度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AOM_Freq">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> ns</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> AOM移频量：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>采样配置</string>
    </attribute>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>采样频率：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sampleFreq">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAfterCurrent</enum>
     </property>
     <item>
      <property name="text">
       <string>400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>266</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>160</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>133</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>114</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>88</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>72</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>探测距离：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_detRange">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> km</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>采样点数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sampleNum">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>96</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>96</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>设置最大单次脉冲数</string>
     </property>
     <property name="text">
      <string>单次脉冲数:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_plsAccNum">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>扫描参数</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> °</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>俯仰角：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_elevationAngle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>11</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>方位角</string>
     </property>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> 步进角：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> °</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_step_azAngle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>81</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> 起始角：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> °</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_start_azAngle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>41</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>定时设置</string>
     </property>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>方向间隔：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_direct_interval">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>圆周间隔：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_circle_interval">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string> min</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>扫描探测</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_circleNum">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_angleNum">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>41</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_anglekey">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>方向数：</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_circlekey">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>圆周数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SP">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>最高转速：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> °/s</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_continousdete">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>连续探测</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>文件存储</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>文件存储路径</string>
     </property>
     <widget class="QPushButton" name="pushButton_pathModify">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>65</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>选择其他文件路径</string>
      </property>
      <property name="text">
       <string>. . .      </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_DatafilePath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>35</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_autocreate_datafile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>65</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>自动创建日期文件夹</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>75</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
